src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -2694,7 +2718,30 @@ impl Coordinator {
 
             // Determine if fast path plan will be used for this explainee
             let fast_path_plan = match explainee {
-                Explainee::Query => peek::create_fast_path_plan(&mut dataflow, GlobalId::Explain)?,
+                Explainee::Query => {
+                    let timestamp_context = self
+                        .sequence_peek_timestamp(
+                            session,
+                            &QueryWhen::Immediately,
+                            cluster_id,
+                            timeline_context,
+                            &id_bundle,
+                            &source_ids,
+                            None, // no real-time recency
```

Comment:
This is somewhat of a lie, if the user has real-time recency enabled, then I'd expect that to be reflected in `EXPLAIN SELECT ... ` and `SELECT ...`.

Probably not a huge deal right now since real-time recency isn't finished.

Reply:
I'll see if I can enable this.

Reply:
> I can either make everything up to `sequence_explain` async

I think this is probably a bad idea, it will do the waiting on the main coord thread which will block everything else from running. 

> leave the code as is.

I think this is the best solution for now as long as we're ok not keeping `EXPLAIN` and `SELECT` in synce.

Reply:
I concur with Joe, but would say it even more strongly: absolutely do not do the above thing by calling await on the recent_timestamp future. If we do need this in the future, it's gotta use the off thread machinery.